t/61 - fix merge fields demo css in export to pdf in production mode

l-lejman opened the pull request:
closes: https://github.com/ckeditor/ckeditor5-demos/issues/61

Due to vite live build limitations i fixed only the prod build export to pdf.
Have to run `vite build` then `vite preview` to see correct export.

f1ames reviewed (changes requested):
> Due to vite live build limitations i fixed only the prod build export to pdf.
> Have to run `vite build` then `vite preview` to see correct export.

Can you expand on that? I recall from the past there was some issue indeed but not sure 🤔 What's the issue and have you checked any workarounds for it?

OTOH, I see you did import styles similar way as it was done before (but from local file not npm package) - https://github.com/ckeditor/ckeditor5-demos/pull/66/files#diff-048830efb6a84ee96a9ce187960ba3b904f18ef37fcc73bd043c1ea64262cecbR97-R100 🤔

l-lejman commented:
> > Due to vite live build limitations i fixed only the prod build export to pdf.
> > Have to run `vite build` then `vite preview` to see correct export.
> 
> Can you expand on that? I recall from the past there was some issue indeed but not sure 🤔 What's the issue and have you checked any workarounds for it?
> 
> OTOH, I see you did import styles similar way as it was done before (but from local file not npm package) - https://github.com/ckeditor/ckeditor5-demos/pull/66/files#diff-048830efb6a84ee96a9ce187960ba3b904f18ef37fcc73bd043c1ea64262cecbR97-R100 🤔

The issue is caused by how vite handles styles imported in js files during hot reload. The only working workaround was to change the build command to disable hot reload but that doesn't make sense since we already have separate commands for building and previewing the production version. Other workarounds, like disabling PostCSS or including CSS files directly in the HTML, didn't work.

I changed the import because with './content.css' vite wasn't copying the file to the dist folder during the production build, which caused styles to be missing in the PDF export.

f1ames reviewed (approved):
@l-lejman thanks for clarification on CSS and PDF export 👍 I see it's also done like that in other demos, so we are consistent here.

LGTM 👍
